src/mono.js

```diff
@@ -34,8 +34,13 @@ const transformMonoMessage = message => {
     const time = predicted.unix();
 
     const journeyStart = parseTime(journeyStartStr);
-    const startDate = journeyStart.format("YYYYMMDD");
-    const startTime = journeyStart.format("HH:mm:ss");
+    let startDate = journeyStart.format("YYYYMMDD");
+    let startTime = journeyStart.format("HH:mm:ss");
+    if (journeyStart.hours() < 4) {
```

Comment:
Looking at our most recent GTFS dump, the last journey of an operating day starts at 28:54:00 i.e. 04:54 normal time. The first journey of an operating day starts at 03:09, so there is overlap. To fix all cases, roisto should send at least the operating day and perhaps also the journey start time expressed in the 30-hour clock.